lib/galaxy/datatypes/binary.py

```diff
@@ -1512,11 +1515,12 @@ def get_config_string(self, filename):
             return config
         except Exception as e:
             log.warning('%s, get model configuration Except: %s', self, e)
-            return ""
+            return "{}"
```

Comment:
Is this a string representation of an empty dictionary? If so, why not just leave the `log.warning` line and let the caller handle `None`? 
Also, out of curiosity, where is this method called from?